bee-node/bee-plugin/bee-plugin-dashboard/src/workers/node_status.rs

```diff
@@ -29,14 +25,18 @@ pub static ALLOCATOR: Cap<alloc::System> = Cap::new(alloc::System, usize::max_va
 
 const NODE_STATUS_METRICS_WORKER_INTERVAL_SEC: u64 = 1;
 
-pub(crate) fn node_status_worker<N>(node: &mut N, users: &WsUsers)
-where
+pub(crate) fn node_status_worker<N>(
+    node: &mut N,
+    node_id: NodeId,
+    node_alias: NodeAlias,
+    bech32_hrp: Bech32Hrp,
+    users: &WsUsers,
+) where
     N: Node,
-    N::Backend: NodeStorageBackend,
+    N::Backend: StorageBackend,
 {
     let tangle = node.resource::<Tangle<N::Backend>>();
     let peer_manager = node.resource::<PeerManager>();
-    let node_config = node.resource::<FullNodeConfig<N::Backend>>();
```

Comment:
Just curious, why can't we do that anymore ?

Reply:
It's in the node crate itself, so it can't be used in the plugin crate without a cyclic dependency.